### PR TITLE
dev-refactor: add new configs to agreement-email-sender

### PR DESCRIPTION
**Pull request by beetlecrunch**

### CODE REVIEW

File: kubernetes/overlays/agreement-email-sender/dev-refactor/deployment.yaml [hunk 1]
```diff
@@ -46,13 +86,13 @@ spec:
             - name: SMTP_USERNAME
               valueFrom:
                 secretKeyRef:
-                  name: smtp-certified
-                  key: PEC_SMTP_USER
+                  name: metrics-reports
```
Comment (beetlecrunch) [inline kubernetes/overlays/agreement-email-sender/dev-refactor/deployment.yaml]: I want to generalize this secret, but in an another PR.

File: kubernetes/overlays/agreement-email-sender/dev-refactor/deployment.yaml [hunk 2]
```diff
@@ -46,13 +86,13 @@ spec:
...
+                  key: SMTP_USER
             - name: SMTP_PASSWORD
               valueFrom:
                 secretKeyRef:
-                  name: smtp-certified
-                  key: PEC_SMTP_PASSWORD
+                  name: metrics-reports
```
Comment (beetlecrunch) [inline kubernetes/overlays/agreement-email-sender/dev-refactor/deployment.yaml]: same

File: kubernetes/configs/dev-refactor/agreement_email_sender.sh [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-AGREEMENT_PEC_SMTP_SECURE=false
+INTEROP_FE_BASE_URL="dev-refactor.selfcare.interop.pagopa.it"
```
Comment (galales) [inline kubernetes/configs/dev-refactor/agreement_email_sender.sh]: isn't this 
```suggestion
INTEROP_FE_BASE_URL="selfcare.refactor.dev.interop.pagopa.it"
```
?

File: kubernetes/overlays/agreement-email-sender/dev-refactor/deployment.yaml (hunk 1 above)
Comment (galales) [reply kubernetes/overlays/agreement-email-sender/dev-refactor/deployment.yaml]: ok, may I ask you to open a task then?

**Review by galales — DISMISSED** [review]

File: kubernetes/overlays/agreement-email-sender/dev-refactor/deployment.yaml (hunk 1 above)
Comment (micdes-pagopa) [reply kubernetes/overlays/agreement-email-sender/dev-refactor/deployment.yaml]: The metrics-report secret contains credentials for the reports sender address, we need a separate K8s secret for the notification address used for agreements (secrets ready on SecretsManager)

File: kubernetes/configs/dev-refactor/agreement_email_sender.sh (hunk 3 above)
Comment (beetlecrunch) [reply kubernetes/configs/dev-refactor/agreement_email_sender.sh]: Yes, thanks!

File: kubernetes/overlays/agreement-email-sender/dev-refactor/deployment.yaml (hunk 1 above)
Comment (beetlecrunch) [reply kubernetes/overlays/agreement-email-sender/dev-refactor/deployment.yaml]: [Here](https://github.com/pagopa/pdnd-interop-platform-deployment/pull/538) the solution for this thread

**Review by micdes-pagopa — APPROVED** [review]

**Pull request merged**